lib/webview/webview.h

```diff
@@ -881,7 +881,7 @@ class win32_edge_engine {
           });
       RegisterClassEx(&wc);
       m_window = CreateWindow(L"Neutralinojs_webview", L"", WS_OVERLAPPEDWINDOW, 99999999,
-                              CW_USEDEFAULT, 640, 480, nullptr, nullptr,
+                              CW_USEDEFAULT, 8000, 8000, nullptr, nullptr,
```

Comment:
Is it necessary to make the window so large?

Reply:
Yes. There is something about this in the description of the Pull Request.
There was an issue otherwise.

Reply:
The issue may stem from creating the window as `WS_OVERLAPPEDWINDOW` see [Win32 docs](https://learn.microsoft.com/en-us/windows/win32/winmsg/window-styles).

It may be worthwhile to call `CreateWindowEx` instead like so:
```c++
m_window = CreateWindowEx(WS_EX_LAYERED, L"Neutralinojs_webview", L"", WS_OVERLAPPEDWINDOW,
                          CW_USEDEFAULT, CW_USEDEFAULT, 640, 480, nullptr, nullptr,
                          GetModuleHandle(nullptr), nullptr);
```

and then enable transparency via `SetLayeredWindowAttributes(m_window, 0, XXX, LWA_ALPHA);` where `XXX` is the alpha value.

Reply:
Yeah, that does work for the entire window and it keeps the resizing behavior consistent, but this also affects the transparency of the webview content, which I think we don't want to happen as we want to control that with CSS instead.

It's a good start though, it's half of a better solution than my proposed PR. We have to be a little clever if we want to keep the window itself transparent, but not the webview.

Reply:
Hmm, would changing
```c++
SetEnvironmentVariable(L"WEBVIEW2_DEFAULT_BACKGROUND_COLOR", L"00FFFFFF");
```
to
```c++
SetEnvironmentVariable(L"WEBVIEW2_DEFAULT_BACKGROUND_COLOR", L"FFFFFFFF");
```
prevent the webview from becoming transparent?

Reply:
Yes, the format for that string is AARRGGBB.